cpp/log/signer.cc

```diff
@@ -59,7 +60,11 @@ std::string Signer::RawSign(const std::string& data) const {
   unsigned int sig_size = EVP_PKEY_size(pkey_.get());
   unsigned char* sig = new unsigned char[sig_size];
 
-  CHECK_EQ(1, EVP_SignFinal(&ctx, sig, &sig_size, pkey_.get()));
+  if (!EVP_SignFinal(&ctx, sig, &sig_size, pkey_.get())) {
+    static char buf[1024];
```

Comment:
I'll have to look around to see if we have another example of getting the error from OpenSSL (I'm pretty sure we do?!?), but I'm having a bad feeling when I see this...